app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
are these used anywhere? Can they be removed? 

Same question for all other values you've added under res/